coa/pkg/apis/v1alpha2/bindings/mqtt/mqtt.go

```diff
@@ -72,21 +72,23 @@ func (m *MQTTBinding) Launch(config MQTTBindingConfig, endpoints []v1alpha2.Endp
 			response = routeTable[request.Route].Handler(request)
 		}
 
-		// needs to carry call-context from request into response
+		// needs to carry request-id from request into response
```

Comment:
Not related to this change - If there are multiple replicas of symphony, will subscribers in mqtt binding of different replicas process the same request multiple times?

Reply:
Indeed. We need to work out the scale-out situation.